### ISSUE TITLE
From RaGOO to RagTag

### ISSUE THREAD

**Issue by PerisD** [malonge/RagTag#3]
Hi Michael,
I tried to use RaGOO. It was interesting to use it and more or less it gave me what I expected. However, my concatenated contigs (I use the option to not break previous scaffolds), the new pseudomolecule, is larger than I expected based on the contigs lengths and the inserted gaps. Anyone is reporting that issue?
`python ragoo.py sample.fasta ref.fasta -g 10000`
For that reason, I moved to RagTag, to see if I can get pseudomolecules with the size I expected. However, I am getting an error
`Traceback (most recent call last):
  File "/software/programs/RagTag/ragtag.py", line 30, in <module>
    from ragtag_utilities.utilities import get_ragtag_version
  File "/software/programs/RagTag/ragtag_utilities/utilities.py", line 37, in <module>
    complements = str.maketrans("ACGTNURYSWKMBVDHacgtnuryswkmbvdh", "TGCANAYRSWMKVBHDtgcanayrswmkvbhd")
AttributeError: type object 'str' has no attribute 'maketrans'
`
Thanks,
Peris

**Comment by malonge** [comment]
Hi Peris,

With respect to your RagTag issue, I assume that happened because you installed with python2 instead of python3. Please correct me if I am wrong and we can dig deeper.

Based on how RagTag turns out, I would be happy to help you with your ragoo results if you are still interested. I think the best thing to do would be to generate an AGP file and just look at what comprises each pseudomolecules.

**Comment by PerisD** [comment]
Hi Michael,
I figure it out the problem with RaGOO. I was using a corrected version of my assembly but when I tried to generate a gff file based on the RaGOO orderings, I used the length of scaffolds from an uncorrected assembly version, for that reason I was getting different lenghts.
About RagTag you are right, as I activate a conda environment with python3 but using Anaconda2 module, the python that is called if I do not set the PATH or I force to used the python in the conda env bin, it always call python 2. So, this is also fixed. 
Thanks!
Peris
